lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -24,7 +23,7 @@ def setUp(self):
             number="B",
             run="C",
         )
-        course_key = SlashSeparatedCourseKey.from_deprecated_string("A/B/C")
+        course_key = self.course.id
         self.instructor = InstructorFactory(course_key=course_key)
```

Comment:
Can we just make this

```
self.instructor = InstructorFactory(course_key=self.course.id)
```

and drop the otherwise unused `course_key` variable?